src/backend/TrafficCourts/Common/Features/Mail/Model/MailTemplateCollection.cs

```diff
@@ -66,6 +66,31 @@ public class MailTemplateCollection
                 HtmlContentTemplate = "TBD",
                 PlainContentTemplate = "TBD"
             },
+        new MailTemplate()
+            {
+                TemplateName = "DisputantUpdateRequestReceivedTemplate",
+                Sender = "DoNotReply@tickets.gov.bc.ca",
+                SubjectTemplate = "TBD (ie. Disputant contact information changes received)",
+                HtmlContentTemplate = "TBD",
+                PlainContentTemplate = "TBD"
+            },
+        new MailTemplate()
```

Comment:
Instead of just leaving TBD text here it has been our practice to put a base message in there which will be refined later by the business.  For this particular email template try Subject "Dispute update requests received."  PlainContentTemplate "Your request to update dispute information was received."

Reply:
Done.

Reply:
I created TCVP-2017 for Colm to fill out for these details